src/html/options.html

```diff
@@ -11,7 +11,7 @@
     <a id="logo" href="ui.html"></a>
   </header>
   <main>
-    <div id="no-bookmarks" data-i18n="whitelist_empty_state" hidden></div>
+    <div id="no-bookmarks" data-i18n="whitelist_empty_state"></div>
```

Comment:
This change should be reverted. Otherwise you introduce the same problem that you want to fix for the empty whitelist for a non-empty whitelist.

Reply:
Correct, my thought was seeing the empty message being replaced might be better. This would be a blank screen which is good too.

Reply:
Updated